### PR TITLE
Changes in up-nrhm reports

### PR DESCRIPTION
**Pull request by lwyszomi**
@snopoke

### CODE REVIEW

File: custom/up_nrhm/reports/asha_facilitators_report.py [hunk 1]
```diff
@@ -3,11 +3,11 @@
 from corehq.apps.reports.standard import CustomProjectReport, DatespanMixin
 from corehq.apps.reports.util import format_datatables_data
 from custom.up_nrhm.sql_data import ASHAFacilitatorsData
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import ugettext_lazy as _, ugettext_noop
 
 
 class ASHAFacilitatorsReport(GenericTabularReport, DatespanMixin, CustomProjectReport):
-    name = _("Format-2 Consolidation of the Functionality numbers")
+    name = ugettext_noop("Format-2 Consolidation of the Functionality numbers")
```
Comment (snopoke) [inline custom/up_nrhm/reports/asha_facilitators_report.py]: Why the change to `ugettext_noop`?

Comment (lwyszomi) [inline custom/up_nrhm/reports/asha_facilitators_report.py]: @snopoke 
When I use 'lazy' then I have: <django.utils.functional.__proxy__ object at 0x724c810>
